packages/venia-ui/lib/components/ProductListing/product.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { useProduct } from '@magento/peregrine/lib/talons/ProductListing/useProduct';
+import { Price } from '@magento/peregrine';
+
+import { mergeClasses } from '../../classify';
+import Kebab from '../MiniCart/kebab';
+import ProductOptions from '../MiniCart/productOptions';
+import Section from '../MiniCart/section';
+import Image from '../Image';
+import defaultClasses from './product.css';
+import { REMOVE_ITEM_MUTATION } from './productListing';
+
+const IMAGE_SIZE = 100;
+
+const Product = props => {
+    const { item } = props;
+    const talonProps = useProduct({
+        item,
+        removeItemMutation: REMOVE_ITEM_MUTATION
+    });
+    const {
+        handleEditItem,
+        handleRemoveFromCart,
+        handleToggleFavorites,
+        isRemoving,
+        product
+    } = talonProps;
+    const { currency, image, name, options, quantity, unitPrice } = product;
+
+    const classes = mergeClasses(defaultClasses, props.classes);
+    const rowMask = isRemoving ? classes.mask : '';
+
+    return (
+        <li className={`${classes.root} ${rowMask}`}>
+            <Image
+                alt={name}
+                classes={{ image: classes.image, root: classes.imageContainer }}
+                width={IMAGE_SIZE}
+                resource={image}
+            />
+            <span className={classes.name}>{name}</span>
+            <ProductOptions
+                options={options}
+                classes={{ options: classes.options, optionLabel: {} }}
+            />
+            <span className={classes.price}>
+                <Price currencyCode={currency} value={unitPrice} />
+                {' ea.'}
+            </span>
+            {/** Quantity Selection to be completed by PWA-119. */}
+            <div className={classes.quantity}>- {quantity} +</div>
+            <Kebab classes={{ root: classes.kebab }}>
+                <Section
+                    text="Move to favorites"
+                    onClick={handleToggleFavorites}
+                    icon="Heart"
+                    isFilled={false}
```

Comment:
Favorites doesn't do anything right now but I think this should use a local state variable at least.

Reply:
Maybe say "wishlist" in code (if not labels too) since that lines up to Magento's terminology.

Reply:
I think for now Tommy is using the existing options kebab which hardcodes the label.

Reply:
I'm going to stick with the copy in the mocks, as I think this functionality is intended to be different from the auth'd wishlist feature. I'll make it usable client-side for @sirugh though 🤗

Reply:
Made this functional client-side in 0ac27f8